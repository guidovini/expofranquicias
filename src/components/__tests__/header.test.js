import React from 'react'
import { render } from '@testing-library/react'

import Header from '../Header'

describe('Header', () => {
  test('Should_RenderHeaderCorrectly', () => {
    const { container } = render(<Header siteTitle="Default Starter" />)
    expect(container).toMatchSnapshot()
  })

  test('Should_RenderHeaderElements', () => {
    const { getByText } = render(<Header siteTitle="Default Starter" />)
    expect(getByText('Expofranquicias Ecuador')).toBeInTheDocument()
    expect(getByText('Franquicias Disponibles')).toBeInTheDocument()
  })
})
