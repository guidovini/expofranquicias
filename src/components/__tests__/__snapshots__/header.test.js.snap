// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should_RenderHeaderCorrectly 1`] = `
<div>
  <nav
    aria-label="main navigation"
    class="navbar is-link"
    role="navigation"
  >
    <div
      class="navbar-brand"
    >
      <a
        class="navbar-item"
        href="/"
      >
        <header
          class="title is-4 has-text-light has-text-weight-light"
        >
          Expofranquicias Ecuador
        </header>
      </a>
      <div
        class="navbar-burger burger"
      >
        <span />
        <span />
        <span />
      </div>
    </div>
    <div
      class="navbar-menu"
    >
      <div
        class="navbar-start"
      />
      <div
        class="navbar-end"
      >
        <a
          class="navbar-item is-size-6"
          href="/"
        >
          <p>
            Inicio
          </p>
        </a>
        <a
          class="navbar-item is-size-6"
          href="/franquicias-disponibles"
        >
          <span
            class="icon"
          >
            <i
              class="fas fa-briefcase"
            />
          </span>
          <span>
            Â 
          </span>
          <span>
            Franquicias Disponibles
          </span>
        </a>
        <a
          class="navbar-item is-size-6"
          href="/contacto"
        >
          Contacto
        </a>
      </div>
    </div>
  </nav>
</div>
`;
